content/blog/page-view-counter/index.md

```diff
@@ -0,0 +1,273 @@
+        ILogger log)
+    {
+        string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+        dynamic data = JsonConvert.DeserializeObject(requestBody);
+        return (ActionResult)new OkObjectResult("0");
+    } 
+```
+
+If we wanted, we could go ahead and publish our project by doing a right-click on the Project and selecting **Publish**. Here is another thing we need to define about our service: How the pricing will be performed. The easiest way, especially if we are using the free Azure account, is to use a Consumption plan, which will give us **1 million** free requests. There are some disadvantages compared to the other 2 plans (Premium and App Services):
+
+- Premium hosting has nice to have features such as perpetual warm instances (as to avoid cold starts which will make our code start with a bit of latency). However, latency is not something we care about in this case.
+- App Service hosting makes our code run on App Service VMs, which makes it optimal if you are using that service already.
+
+Lastly, when publishing to a consumption plan, it asks us for the following:
+
+- Name: The name of the plan.
+- Subscription: After authenticating with your account, your free sub will appear here.
+- Resource group: Just a way to group resources in Azure. All resources need to be deployed into one. You will need to create a new one if you didn't already
+- Location: The datacenter in which you want your Function to run.
+- Azure Storage: The plan needs to store metadata and for this, they use a storage account. You will need to create one.
+
+Notice that the last field is an Azure Storage Account. This is convenient for us because as we said, we will be using Storage tables and this saves us the creation of one. Let's create the Storage Account and proceed with the publishing. A Storage Account name must be unique because it creates public endpoints through which you can access the service (e.g. *https://your-storage-name.table.core.windows.net*). You are charged by the data egress and ingress, so you can create as many as you want.
+
+You can also access the [Azure Portal](https://portal.azure.com/), log in and see the resource group with the resources you just created!
+
+### Using Azure Storage
+
+Azure Storage encompasses many services such as Blobs, Tables, Queues, Files, etc. In our case, we mentioned we will use Tables because of the easy key-value storage. We can think of a page URL as the key, and the times it was counted as the value that we want to retrieve.
+
+At this point, we want to install the Azure Storage Tables NuGet package (or the equivalent for the language you want). We are actually going to use the [Cosmos DB Tables API](https://docs.microsoft.com/en-us/azure/cosmos-db/tutorial-develop-table-dotnet#install-the-required-nuget-package), since it includes the API for accessing storage tables.
+Let's add the reference to it:
```

Comment:
love to have the name of the file where we are adding this `using` statement, I guess is the file we have been working on but might be a good a idea to give it a name consistently